packages/gatsby-plugin-coffeescript/README.md

```diff
@@ -0,0 +1,40 @@
+# gatsby-plugin-coffeescript
+Provides drop-in support for CoffeeScript and CJSX.
+
+## Install
+`yarn add gatsby-plugin-coffeescript`
+
+## How to use
+1. Include the plugin in your `gatsby-config.js` file.
+2. Write your components in CJSX or CoffeeScript.
+
+```javascript
+// in gatsby-config.js
+plugins: [
+  // no configuration
+  `gatsby-plugin-coffeescript`,
+  // custom configuration
+  {
+    resolve: `gatsby-plugin-coffeescript`,
+    // options are passed directly to the compiler
+    options: {}
+  }
+]
+```
+
+## Notes
+First, note that CoffeeScript + React is a troubled combination. This plugin
+relies upon deprecated modules that may someday prove to be dysfunctional or
+otherwise deficient.
+
+Furthermore, note that the installed version of CoffeeScript is @next. This is 
+not optional - *named exports are required for page queries to work properly.*
+
+You will need to manually edit your `coffee-loader` installation and install 
```

Comment:
It's an issue with the `coffee-loader` package - it's not (yet) set up to use 2.x (which we need for imports/exports). There's a recently-submitted PR that adds support, so this line of documentation will only be relevant for a couple days.

The repo with the PR could be installed to the project directory as `jalavosus:cs2`, and it should all work. ...that's probably be a better solution, albeit one I didn't think of until now.

https://github.com/webpack-contrib/coffee-loader/issues/33
https://github.com/webpack-contrib/coffee-loader/pull/30